Assignment1

```diff
@@ -0,0 +1,38 @@
+#include<iostream>
+using namespace std;
+int Array_Size;
+__global__ void Sum(int* d1_in, int* d2_in, int* d_out)
+{
+    int i = blockIdx.x * blockDim.x + threadIdx.x;
+        d_out[i] = d1_in[i] + d2_in[i];
+}
+int main()
+{
+    cout << "Enter the array size : ";
+    cin >> Array_Size;
+    int h1_in[Array_Size], h_out[Array_Size],h2_in[Array_Size];
+    int Array_Bytes = Array_Size * sizeof(int);  
+    cout<<"Enter "<<Array_Size<<" elements(Array1)\n";
+    for(int i=0; i<Array_Size; i++)
+    cin>>h1_in[i];
+    cout<<"Enter "<<Array_Size<<" elements(Array2)\n";
+    for(int i=0; i<Array_Size; i++)
+    cin>>h2_in[i];
```

Comment:
Consider generating your inputs. Because we want to use GPUs on larger inputs. You can do it randomly or by some logic.